syllabus/7-Polkadot/README.md

```diff
@@ -2,22 +2,66 @@
 
 ## Learning Outcomes
 
+* Mission
+    * What are the goals behind Polkadot?
+* Elections
+    * How do we get validators to play our game?
+    * NPoS
+* Consenus
+    * How validators play the consensus game?
+    * Data collation
+    * Availability sharding
+    * Execution sharding
+    * Scheduling (Coretime)
+    * XCMP
+* Governance
+    * How does the system evolve?
+    * OpenGov
+    * Fellowship
+* Tooling
+    * What extra tools are available to build on Polkadot?
+    * Cumulus
+    * Zombienet
+
 ## Hands on activities
 
-## Schedule
+- Build a parachain (maybe)
+- Malicious validator zombienet testing (maybe)
+- Quiz
+
+# Schedule
 
 ## Day 1
 
 ### Morning
 
+- Introduction to Polkadot part 2 [Maciej]
+- Data Preparation - Collation & Backing [Maciej]
+- Data Sharding - Data Availability [Bradley]
+- Execution Sharding - Approvals and Disputes [Maciej]
+
 ### Afternoon
 
+- Coretime [Maciej]
+- Cumulus [Bradley]
```

Comment:
```suggestion
- Cumulus Deep Dive [Bradley]
```

(noting that an intro to cumulus should come much sooner)

Reply:
This will be cumulus deep dive. I hope intro takes place in sdk